docs/building.md

```diff
@@ -105,7 +105,6 @@ Configure maven to use the native windows maven repository - then, when doing `.
 
 ## Building for Firefox iOS
```

Comment:
Yeah I think our docs in general on iOS can use a full re-haul (there  are a few sections referencing that "Carthage will *soon* be gone"), but we can do that as a followup